app/views/forms/edit/abs/edit-absPermit.js

```diff
@@ -83,7 +83,8 @@ import 'views/forms/view/abs/view-abs-permit.directive';
               fieldQueries : [],
               searchText: searchText
           }
-          //queryOptions.identifier = (($scope.document||{​​​​​}​​​​​).header||{​​​​​}​​​​​).identifier
+           if($scope.document?.header.identifier)
+               queryOptions.fieldQueries = ["NOT (identifier_s:" + solr.escape($scope.document.header.identifier)+ ")"];
```

Comment:
We need to start using new syntax as possible
```suggestion
               queryOptions.fieldQueries = [`NOT identifier_s:${solr.escape($scope.document.header.identifier)}`];
```